src/test/kotlin/io/github/mojira/arisa/modules/commands/RemoveUserCommandTest.kt

```diff
@@ -0,0 +1,54 @@
+package io.github.mojira.arisa.modules.commands
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import io.github.mojira.arisa.utils.mockIssue
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+import java.lang.RuntimeException
+
+class RemoveUserCommandTest : StringSpec({
+    "should query user activity and post a comment with all tickets" {
+        var calledSearch = false
+        var comment: String? = null
+        var commentRestrictions: String? = null
+        val queriedIssues = mutableListOf<String>()
+
+        val issueList = listOf("MC-1", "MC-12", "MC-1234", "MC-12345", "MC-1")
+
+        val command = RemoveUserCommand(
+            { _, _ ->
+                Either.fx {
+                    calledSearch = true
+                    issueList
+                }
+            },
+            { ticketId ->
+                Either.fx {
+                    queriedIssues.add(ticketId)
+                    throw RuntimeException("Cannot mock a rcarz issue")
```

Comment:
We cannot? mock(JiraIssue.class) doesnt work?

Reply:
I couldn't figure out how it would work. And I don't know what `mock(JiraIssue.class)` would be? `JiraIssue` isn't a class and `mock` isn't a function that I know of.

All the fields in the rcarz issue are private so I wouldn't know how to mock them and I don't want to waste time writing a mock for a library that we won't use in the future anyway. And it'd be good to refactor this anyway so that I don't even need to worry about rcarz issues.

Reply:
Aaah I think I've found what you mean. Brb doing that.

Reply:
Ok but this is still very annoying to mock even with mockK. I guess we're just gonna need to have this test incomplete for now.